lib/utilities/lint-fix.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const fs = require('fs-extra');
+const execa = require('../utilities/execa');
+
+async function run(ui) {
+  let lintFixScriptName = 'lint:fix';
+  let cwd = process.cwd();
+  let packageJson = fs.readJsonSync('package.json');
+
+  let hasLintFixScript = !!packageJson.scripts[lintFixScriptName];
+
+  if (!hasLintFixScript) {
+    ui.writeWarnLine(
+      'Lint fix skipped: "lint:fix" not found in package.json scripts. New files might not pass linting.'
+    );
+
+    throw new Error('"lint:fix" not found in package.json scripts');
```

Comment:
Just to clarify, we expect this to be absorbed by the `try`/`catch` up in `lib/tasks/install-blueprint.js` and `lib/tasks/generate-from-blueprint.js` right? 

We definitely don't want to bubble out an error just because folks don't have `lint:fix` defined...

Reply:
That's correct, however I wasn't keen on it so I replaced the error throw with a `return`.